docs/pages/modules/use-create-expo-module-in-your-project.mdx

```diff
@@ -0,0 +1,180 @@
+
+If you're not familiar with how to configure your project as a monorepo, check the **[Working with monorepos](/guides/monorepos/)** guide.
+
+### 1. Initialize a new module
+
+Once you have set up the basic monorepo structure, create a new module using `create-expo-module`:
+
+<Terminal cmd={['$ npx create-expo-module packages/expo-settings']} />
+
+### 2. Set up our workspace
+
+We won't need the **example** folder included by `create-expo-module`. Let's clean up the default module with the following command:
+
+<Terminal cmd={['$ rm -rf packages/expo-settings/example']} />
+
+Now, let's configure `autolinking` so all of our apps can use the newly created module. Add the following block to the **package.json** file of each app under the **apps** folder:
+
+```json package.json
+"expo": {
+  "autolinking": {
+    "nativeModulesDir": "../../packages"
+  }
+}
+```
+
+### 3. Running your module
+
+Now let's run an app to ensure everything is working. Start the TypeScript compiler inside **packages/expo-settings** to watch for changes and rebuild the module JavaScript.
+
+<Terminal
+  cmdCopy="cd packages/expo-settings && npm run build"
+  cmd={['$ cd packages/expo-settings', '$ npm run build']}
+/>
+
+In another terminal window, choose one of your apps under the **apps** folder and run the `prebuild` command. You will need to do this in all apps inside your monorepo in order to use the new module.
+
+<Terminal cmd={['$ npx expo prebuild --clean']} />
+
+Compile and run the app with the following command:
+
+<Terminal
+  cmdCopy="npx expo run:ios"
+  cmd={[
+    '# Run the app on Android',
+    '$ npx expo run:android',
+    '# Run the app on iOS',
+    '$ npx expo run:ios',
+  ]}
+/>
+
+You should now be able to use the module inside your app. To test this, let's edit the **App.js**
+file in the app and render the `hello` text from our `expo-settings` module:
+
+```javascript App.js
+import * as Settings from 'expo-settings';
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export default function App() {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>{Settings.hello()}</Text>
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+```
+
+If you've configured everything correctly you should see a text saying "Hello world! 👋” in your app.
+
+## Publishing your module
+
+If you don't want to go through the process of setting up a monorepo, you can also publish your module to npm and just add it to your projects as a normal dependency.
+
+### 1. Initialize a new module
+
+First, start by creating a new module using `create-expo-module`. Pay attention to each prompt as we're going to publish this library, so make sure to choose a unique name for your npm package.
+
+<Terminal cmd={['$ npx create-expo-module expo-settings']} />
+
+### **2. Run the example project**
```

Comment:
```suggestion
### 2. Run the example project
```

No need to bold the heading of a section. Doesn't add any emphasis or significance in when the markdown content is rendered.

Reply:
No worries!